postr/youtube_postr.py

```diff
@@ -100,12 +126,24 @@ def get_user_followers(self, text: str) -> List[str]:
     def remove_post(self, post_id: str) -> bool:
         ''' This method removes the post with the specified id
         and returns the success of this action'''
+        # post_id in this case should be the upload_id of the video/the video's id
         videos_delete(self.build, id=post_id)
         return True
 
 # Remove keyword arguments that are not set
 
 
+def playlist_items_list_by_playlist_id(client: Any, **kwargs: Any) -> Any:
+    # See full sample for function
+    kwargs = remove_empty_kwargs(**kwargs)
+
```

Comment:
smells like repeated code but it's only 3 lines tbh, 2 if you remove the unnecessary assign